tests/host_nodes_test.py

```diff
@@ -0,0 +1,10 @@
+
+from htcanalyze.host_nodes import node_cache
+
+
+def test_node_cache():
+    node_cache.gethostbyaddrcached("172.217.0.0")
+    assert node_cache.rdns_cache["172.217.0.0"] == "ord38s04-in-f0.1e100.net"
+    node_cache.gethostbyaddrcached("172.217.0.0")  # lookup again
+    node_cache.gethostbyaddrcached("NoIP")
+    assert node_cache.rdns_cache["NoIP"] == "NoIP"
```

Comment:
Minor nitpick: no newline at the end of the file.

Reply:
Wait what ? This is a common fault created by pep8/pycodestyle. From this tool it'S recommended to have one empty newline at the end of a file. So in this case I will stick with pep8 to have it run clear.